user-guide/Cloud_Platform/AboutCloudPlatform/Connecting_your_DataMiner_System_to_the_cloud.md

```diff
@@ -27,7 +27,7 @@ To connect your DMS to the cloud:
    > - If Cloud Gateway version v2.9.4 or higher is installed, you can check whether your network complies with the requirements for the cloud platform using the *ConnectionTester.exe* tool from the folder `Program files\Skyline Communications\Dataminer CloudGateway\`.
    > - For more information about cloud connectivity and security, see [Connecting to the DataMiner Cloud Platform](xref:Cloud_connectivity_and_security#connecting-to-the-dataminer-cloud-platform).
 
-1. Download the latest DataMiner **Cloud Pack installer** from [DataMiner Dojo](https://community.dataminer.services/downloads/) and install it on one or more DMAs in the cluster. If you are on the Feature Release track, .NET 5 is already included in DataMiner from version 10.1.12 onwards, so select the package without .NET 5. If you are on the Main Release track, .NET 5 is only included from version 10.2.0 \[CU4] onwards, so for earlier 10.2.0 versions, you will need to select the package with .NET 5.
+1. Download the latest DataMiner **Cloud Pack installer** from [DataMiner Dojo](https://community.dataminer.services/downloads/) and install it on one or more DMAs in the cluster. More information on the contents of the Cloud Pack can be found [here](xref:CloudPack). If you are on the Feature Release track, .NET 5 is already included in DataMiner from version 10.1.12 onwards, so select the package without .NET 5. If you are on the Main Release track, .NET 5 is only included from version 10.2.0 \[CU4] onwards, so for earlier 10.2.0 versions, you will need to select the package with .NET 5.
```

Comment:
so you may select the package wihtout .NET 5 ?

Reply:
yes if it is already installed